meson_options.txt

```diff
@@ -10,4 +10,4 @@ option('JSON_ABSOLUTE_PATH_PREFIX', type: 'string', value: '/usr/share/vpd/',  d
 option('SYSTEM_VPD_FILE_PATH', type: 'string', value: '/sys/bus/i2c/drivers/at24/8-0050/eeprom',  description: 'EEPROM path of system VPD.')
 option('VPD_SYMLIMK_PATH', type: 'string', value: '/var/lib/vpd',  description: 'Symlink folder for VPD invnetory JSONs')
 option('PIM_PATH_PREFIX', type: 'string', value: '/xyz/openbmc_project/inventory', description: 'Prefix for PIM inventory paths.')
-option('ibm_system', type: 'feature', value : 'disabled', description: 'Enable code specific to IBM systems.')
\ No newline at end of file
+option('ibm_system', type: 'feature', value : 'enabled', description: 'Enable code specific to IBM systems.')
```

Comment:
You need to make the change in recipe file.

Reply:
The intention of this PR is to get the `ibm-openbmc` CI job to have `ibm_system` enabled. The `ibm-openbmc` CI job picks up options from `meson_options.txt`. How will changing the `bitbake` recipe file help?

Reply:
Ahh, ok. In that case we can keep it till we push the code upstream.